tests/test_jax_nonbonded.py

```diff
@@ -158,17 +155,24 @@ def generate_random_inputs(n_atoms, dim, instance_flags=difficult_instance_flags
     min_dist = 0.1
     conf, box = resolve_clashes(conf, box, min_dist=min_dist)
 
+    cutoff = 1.2
+    if instance_flags["randomize_cutoff"]:
+        cutoff += rand()
```

Comment:
note that we have hard requirement that cutoff must be less than half of the box width (not sure what the box sizes here are).

Reply:
It doesn't seem like we currently enforce this requirement in the reference potentials(?) (note this test code is for the reference nonbonded potential)

(also, this was not added in the PR but moved up from lower in the file so that the randomized cutoff could be used to scale the random w coordinates)